packages/node/src/protocol/uninstall-virtual-app.ts

```diff
@@ -114,6 +114,7 @@ export const UNINSTALL_VIRTUAL_APP_PROTOCOL: ProtocolExecutionFlow = {
       Opcode.IO_SEND_AND_WAIT,
       {
         // m4
+        ...context.message,
```

Comment:
why is this here?

Reply:
Because when `...context.message` was removed last time it did not correctly add in every explicit parameter it was responsible for adding; this lead to `undefined` protocol messages being passed around.